src/System.Private.CoreLib/shared/System/Numerics/Vector.cs

```diff
@@ -75,9 +75,8 @@ public static int Count
         public static Vector<T> Zero
         {
             [Intrinsic]
-            get => s_zero;
+            get => default;
         }
-        private static readonly Vector<T> s_zero = new Vector<T>();
```

Comment:
@tannergooding, is this change ok?  This field seems unnecessary, but given get_Zero is an intrinsic, I wasn't sure if there was something special happening here.

Reply:
It's okay, but we should also be calling `ThrowHelper.ThrowForUnsupportedVectorBaseType<T>();`

Vector<T> explicitly only supports `byte`, `double`, `short`, `int`, `long`, `sbyte`, `float`, `ushort`, `uint`, and `ulong` and throws for others with the hardware accelerated path:

![image](https://user-images.githubusercontent.com/10487869/68341535-85a0e100-009d-11ea-801e-ad86531fd2e0.png)

CC. @CarolEidt

Reply:
Sounds like a fine thing to fix, but I'd like it to be done separately from this PR, which is entirely style.